resources/views/success.blade.php

```diff
@@ -0,0 +1,63 @@
+
+@extends('layouts.app')
+
+@section('title', 'Course registered')
+
+@section('content')
+<div class="padded content">
+	<h1>Success! Your course was registered.</h1>
+	<div>
+		<h2>Overview:</h2>
+		<p>Course name: Cool course name</p>
+		<p>Teacher(s):</p>
+		<ul>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+		</ul>
+		<p>Course semester: Fall 2018</p>
+		<p>Course type: Client</p>
+		<p>Course schedule: Monday, Thursday</p>
+		<p>Starting time: 10:30 AM</p>
+		<p>Teams of: 4</p>
+		<p>Affiliated courses:</p>
+		<ul>
+			<li>Random</li>
+			<li>Random</li>
+			<li>Random</li>
+			<li>Random</li>
+		</ul>
+		<p>Your course key is ABCDEF.</p>
+	</div>
+</div>
+
+
+	{{-- <!-- Esto es lo que iría con la respuesta del backend -->
+	<div>
+		<h2>Overview:</h2>
+		<p>Course name: {{ $courseName }}</p>
+		<p>Teacher(s):</p>
+		<ul>
+		@foreach (teachers as teacher)
+			<li>{{ $teacher }}</li>
+		@endforeach
+		</ul>
+		<p>Course semester: {{ $courseSemester }}</p>
+		<p>Course type: {{ $courseType }}</p>
+		<p>Course schedule: {{ $courseSchedule }}</p>
```

Comment:
¿Todos los fields son required? ¿Ya consideraste el caso en donde no todos están poblados? ¿Cómo se ve eso?

Reply:
Todos menos 'Associated courses' son obligatorios, solo en ese se vería un vacío, así:
<img width="631" alt="screen shot 2019-03-01 at 12 14 06 am" src="https://user-images.githubusercontent.com/8518253/53619990-f90d5d00-3bb6-11e9-92ed-fa356b632e35.png">

Reply:
Ahora que lo veo creo que no vale la pena hacer un `success` y un `course details` view por separado. Cuando dije success screen me refería más al course detail page porque eso en esencia indica success, y como como quiera necesitamos la de course detail igual y vale la pena mover esto a course detail (a lo mejor con un Flash message) que le diga que Successsfully registered course. y usar ambas (con esta misma información que ya está aquí) para ambos propósitos.

Reply:
Also, aquí me confunde que _your course key..._ esté en un formato diferente. Como no tiene associated courses se ve como si ese fuera un label para "your course key...". El course key me parece también que debería ser mucho más prominente (maybe del lado derecho superior para que sea easily retrievable).

Reply:
Also, consider using code styling: `<code>823FAN</code>` & styling it in a monospaced font.

Reply:
Done! 😄